<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>820</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>820</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>820</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>TS3500</string>
  </property>
  <property name="whatsThis">
   <string/>
  </property>
  <property name="inputMethodHints">
   <set>Qt::ImhNone</set>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="minimumSize">
    <size>
     <width>820</width>
     <height>600</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>820</width>
     <height>600</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">alternate-background-color: rgb(77, 92, 147);
opacity: 0.5;</string>
   </property>
   <widget class="QWidget" name="gridLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>250</y>
      <width>160</width>
      <height>321</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="2" column="0">
      <widget class="QPushButton" name="BtnPrivt2">
       <property name="text">
        <string>A23031</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QPushButton" name="BtnPrivt4">
       <property name="text">
        <string>A23033</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QPushButton" name="BtnPrivt6">
       <property name="text">
        <string>A23035</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="BtnPrivt1">
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>A23030</string>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <widget class="QPushButton" name="BtnPrivt7">
       <property name="text">
        <string>A23036</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QPushButton" name="BtnPrivt5">
       <property name="text">
        <string>A23034</string>
       </property>
      </widget>
     </item>
     <item row="10" column="0">
      <widget class="QPushButton" name="BtnPrivt9">
       <property name="text">
        <string>A23039</string>
       </property>
      </widget>
     </item>
     <item row="9" column="0">
      <widget class="QPushButton" name="BtnPrivt8">
       <property name="text">
        <string>A23038</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QPushButton" name="BtnPrivt3">
       <property name="text">
        <string>A23032</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>230</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:12pt; font-weight:600; font-style:italic; vertical-align:super;&quot;&gt;Fitas HSM PRIVATE&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;PRIVATE TAPES&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>230</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:12pt; font-weight:600; font-style:italic; vertical-align:super;&quot;&gt;Fitas HSM PRIVATE&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;MIGRATE TAPES&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>230</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:12pt; font-weight:600; font-style:italic; vertical-align:super;&quot;&gt;Fitas HSM PRIVATE&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;DASD TAPES&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_4">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>250</y>
      <width>160</width>
      <height>321</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_4">
     <item row="10" column="0">
      <widget class="QPushButton" name="BtnMig8">
       <property name="text">
        <string>MIG007</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QPushButton" name="BtnMig3">
       <property name="text">
        <string>MIG002</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QPushButton" name="BtnMig5">
       <property name="text">
        <string>MIG004</string>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <widget class="QPushButton" name="BtnMig6">
       <property name="text">
        <string>MIG005</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QPushButton" name="BtnMig4">
       <property name="text">
        <string>MIG003</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QPushButton" name="BtnMig2">
       <property name="text">
        <string>MIG001</string>
       </property>
      </widget>
     </item>
     <item row="9" column="0">
      <widget class="QPushButton" name="BtnMig7">
       <property name="text">
        <string>MIG006</string>
       </property>
      </widget>
     </item>
     <item row="11" column="0">
      <widget class="QPushButton" name="BtnMig9">
       <property name="text">
        <string>MIG008</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QPushButton" name="BtnMig1">
       <property name="text">
        <string>MIG000</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_5">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>250</y>
      <width>160</width>
      <height>321</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_5">
     <item row="11" column="0">
      <widget class="QPushButton" name="BtnDasd7">
       <property name="text">
        <string>A00006</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QPushButton" name="BtnDasd3">
       <property name="text">
        <string>A00002</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="BtnDasd1">
       <property name="whatsThis">
        <string/>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>A00000</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QPushButton" name="BtnDasd2">
       <property name="text">
        <string>A00001</string>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <widget class="QPushButton" name="BtnDasd5">
       <property name="text">
        <string>A00004</string>
       </property>
      </widget>
     </item>
     <item row="9" column="0">
      <widget class="QPushButton" name="BtnDasd6">
       <property name="text">
        <string>A00005</string>
       </property>
      </widget>
     </item>
     <item row="13" column="0">
      <widget class="QPushButton" name="BtnDasd9">
       <property name="text">
        <string>A00008</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QPushButton" name="BtnDasd4">
       <property name="text">
        <string>A00003</string>
       </property>
      </widget>
     </item>
     <item row="12" column="0">
      <widget class="QPushButton" name="BtnDasd8">
       <property name="text">
        <string>A00007</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-1</y>
      <width>820</width>
      <height>600</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>820</width>
      <height>600</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>800</width>
      <height>600</height>
     </size>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <widget class="QWidget" name="gridLayoutWidget_6">
     <property name="geometry">
      <rect>
       <x>610</x>
       <y>480</y>
       <width>181</width>
       <height>81</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_6">
      <item row="4" column="0">
       <widget class="QPushButton" name="BtnDasd">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(203, 203, 203);</string>
        </property>
        <property name="text">
         <string>DASD</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QPushButton" name="BtnMig">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(203, 203, 203);</string>
        </property>
        <property name="text">
         <string>Migrate</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QPushButton" name="BtnOff">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(203, 203, 203);</string>
        </property>
        <property name="text">
         <string>Off</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="BtnPrivate">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(203, 203, 203);</string>
        </property>
        <property name="text">
         <string>Private</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QLabel" name="TxtCriaTape">
        <property name="text">
         <string>Cadastrar novas fitas  </string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="gridLayoutWidget">
     <property name="geometry">
      <rect>
       <x>610</x>
       <y>290</y>
       <width>182</width>
       <height>71</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="verticalSpacing">
       <number>0</number>
      </property>
      <item row="2" column="0" alignment="Qt::AlignLeft">
       <widget class="QLabel" name="label_4">
        <property name="maximumSize">
         <size>
          <width>180</width>
          <height>25</height>
         </size>
        </property>
        <property name="text">
         <string>Mount Tape Status</string>
        </property>
        <property name="textFormat">
         <enum>Qt::AutoText</enum>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="textInteractionFlags">
         <set>Qt::NoTextInteraction</set>
        </property>
       </widget>
      </item>
      <item row="3" column="0" alignment="Qt::AlignLeft|Qt::AlignVCenter">
       <widget class="QTextBrowser" name="TxtLCD">
        <property name="maximumSize">
         <size>
          <width>250</width>
          <height>50</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 9pt &quot;Courier New&quot;;
background-color: qlineargradient(spread:pad, x1:0.483, y1:0.335, x2:1, y2:1, stop:0 rgba(250, 250, 250, 255), stop:1 rgba(215, 216, 207, 255));</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QSlider" name="verticalSlider">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>250</y>
       <width>16</width>
       <height>321</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QProgressBar" name="progressBar">
     <property name="geometry">
      <rect>
       <x>610</x>
       <y>250</y>
       <width>181</width>
       <height>16</height>
      </rect>
     </property>
     <property name="baseSize">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
     <property name="palette">
      <palette>
       <active>
        <colorrole role="Button">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>181</red>
           <green>148</green>
           <blue>148</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Base">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>181</red>
           <green>148</green>
           <blue>148</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Window">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>181</red>
           <green>148</green>
           <blue>148</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="AlternateBase">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>77</red>
           <green>92</green>
           <blue>147</blue>
          </color>
         </brush>
        </colorrole>
       </active>
       <inactive>
        <colorrole role="Button">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>181</red>
           <green>148</green>
           <blue>148</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Base">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>181</red>
           <green>148</green>
           <blue>148</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Window">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>181</red>
           <green>148</green>
           <blue>148</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="AlternateBase">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>77</red>
           <green>92</green>
           <blue>147</blue>
          </color>
         </brush>
        </colorrole>
       </inactive>
       <disabled>
        <colorrole role="Button">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>181</red>
           <green>148</green>
           <blue>148</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Base">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>181</red>
           <green>148</green>
           <blue>148</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Window">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>181</red>
           <green>148</green>
           <blue>148</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="AlternateBase">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>77</red>
           <green>92</green>
           <blue>147</blue>
          </color>
         </brush>
        </colorrole>
       </disabled>
      </palette>
     </property>
     <property name="cursor">
      <cursorShape>WaitCursor</cursorShape>
     </property>
     <property name="focusPolicy">
      <enum>Qt::WheelFocus</enum>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">QProgressBar{
	background-color: rgb(181, 148, 148);
}

QProgressBar::chunk{
background-color: qlineargradient(spread:pad, x1:0, y1:0.136, x2:1, y2:0.125, stop:0.375 rgba(161, 179, 191, 255), stop:1 rgba(94, 133, 144, 255));
}</string>
     </property>
     <property name="value">
      <number>0</number>
     </property>
     <property name="textVisible">
      <bool>false</bool>
     </property>
     <property name="invertedAppearance">
      <bool>false</bool>
     </property>
     <property name="format">
      <string/>
     </property>
    </widget>
    <widget class="QTextBrowser" name="textBrowser">
     <property name="geometry">
      <rect>
       <x>17</x>
       <y>10</y>
       <width>781</width>
       <height>201</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>850</width>
       <height>280</height>
      </size>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">QTextBrowser{
	background-color: rgb(236, 236, 236);
font: 9pt &quot;SansSerif&quot;;
font: 9pt &quot;Courier New&quot;;
text-align: justify;
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="sizeAdjustPolicy">
      <enum>QAbstractScrollArea::AdjustToContentsOnFirstShow</enum>
     </property>
     <property name="autoFormatting">
      <set>QTextEdit::AutoAll</set>
     </property>
    </widget>
    <widget class="QWidget" name="gridLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>610</x>
       <y>410</y>
       <width>181</width>
       <height>31</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="1">
       <widget class="QLineEdit" name="Mdriver">
        <property name="maximumSize">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(231, 231, 231);
opacity: 0.5;</string>
        </property>
        <property name="placeholderText">
         <string>Driver</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QPushButton" name="BtnManual">
        <property name="maximumSize">
         <size>
          <width>40</width>
          <height>30</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>7</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(212, 212, 212);</string>
        </property>
        <property name="text">
         <string>OK</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLineEdit" name="Mtape">
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>20</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(231, 231, 231);
opacity: 0.5;</string>
        </property>
        <property name="inputMethodHints">
         <set>Qt::ImhUppercaseOnly</set>
        </property>
        <property name="placeholderText">
         <string>Fita</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>610</x>
       <y>380</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 8pt &quot;Segoe UI&quot;;</string>
     </property>
     <property name="text">
      <string>Informe o volser e driver da fita.</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignJustify|Qt::AlignVCenter</set>
     </property>
    </widget>
   </widget>
   <zorder>frame</zorder>
   <zorder>gridLayoutWidget_3</zorder>
   <zorder>label</zorder>
   <zorder>label_2</zorder>
   <zorder>label_3</zorder>
   <zorder>gridLayoutWidget_4</zorder>
   <zorder>gridLayoutWidget_5</zorder>
  </widget>
 </widget>
 <tabstops>
  <tabstop>BtnPrivt1</tabstop>
  <tabstop>BtnPrivt2</tabstop>
  <tabstop>BtnPrivt3</tabstop>
  <tabstop>BtnPrivt4</tabstop>
  <tabstop>BtnPrivt5</tabstop>
  <tabstop>BtnPrivt6</tabstop>
  <tabstop>BtnPrivt7</tabstop>
  <tabstop>BtnPrivt8</tabstop>
  <tabstop>BtnPrivt9</tabstop>
  <tabstop>BtnMig1</tabstop>
  <tabstop>BtnMig2</tabstop>
  <tabstop>BtnMig3</tabstop>
  <tabstop>BtnMig4</tabstop>
  <tabstop>BtnMig5</tabstop>
  <tabstop>BtnMig6</tabstop>
  <tabstop>BtnMig7</tabstop>
  <tabstop>BtnMig8</tabstop>
  <tabstop>BtnMig9</tabstop>
  <tabstop>BtnDasd1</tabstop>
  <tabstop>BtnDasd2</tabstop>
  <tabstop>BtnDasd3</tabstop>
  <tabstop>BtnDasd4</tabstop>
  <tabstop>BtnDasd5</tabstop>
  <tabstop>BtnDasd6</tabstop>
  <tabstop>BtnDasd7</tabstop>
  <tabstop>BtnDasd8</tabstop>
  <tabstop>BtnDasd9</tabstop>
  <tabstop>verticalSlider</tabstop>
  <tabstop>progressBar</tabstop>
  <tabstop>TxtLCD</tabstop>
  <tabstop>Mtape</tabstop>
  <tabstop>Mdriver</tabstop>
  <tabstop>BtnManual</tabstop>
  <tabstop>BtnMig</tabstop>
  <tabstop>BtnPrivate</tabstop>
  <tabstop>BtnDasd</tabstop>
  <tabstop>BtnOff</tabstop>
  <tabstop>textBrowser</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>BtnOff</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>789</x>
     <y>579</y>
    </hint>
    <hint type="destinationlabel">
     <x>409</x>
     <y>296</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>progressBar</sender>
   <signal>valueChanged(int)</signal>
   <receiver>textBrowser</receiver>
   <slot>forward()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>704</x>
     <y>272</y>
    </hint>
    <hint type="destinationlabel">
     <x>691</x>
     <y>128</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>textBrowser</sender>
   <signal>cursorPositionChanged()</signal>
   <receiver>progressBar</receiver>
   <slot>raise()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>686</x>
     <y>80</y>
    </hint>
    <hint type="destinationlabel">
     <x>660</x>
     <y>306</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
